demo-drone-app/src/components/Drones.vue

```diff
@@ -122,7 +122,37 @@
             </template>
             <template slot="expand" slot-scope="props">
               <v-card flat>
-                <v-card-text>{{props.item.description}}</v-card-text>
+                <v-card-text v-if="!editDescription">
+                  <v-layout row justify-space-between>
+                    <v-flex xs8>
+                      {{props.item.description}}
+                    </v-flex>
+                    <v-flex xs2>
+                      <v-btn flat @click="editDescription=true">
+                        <v-icon>edit</v-icon>
+                      </v-btn>
+                    </v-flex>
+                  </v-layout>
+                </v-card-text>
+                <v-card-text v-if="editDescription">
+                  <v-layout row justify-space-between>
+                    <v-flex xs8>
+                      <v-text-field
```

Comment:
Do we want to have the functionality to edit any other part of the drone like manufacturing, color, etc? Honestly was not sure since this was just kind of open-ended, curious what you think.